docs/dev-guides/intellij.md

```diff
@@ -0,0 +1,196 @@
+# IntelliJ for Power Users
+IntelliJ has many secrets, and the easiest way to discover them is on accident. However, it's worth
+compiling some of the most useful ones in one place, so that one can benefit more quickly from them.
+All keyboard shortcuts are  9-5for the MacOS scheme.
+
+## Editing
+### Autocomplete
+When typing symbols, auto-completion is available. Type tab then Enter to accept
+the highlighted term, or use up and down keys to choose another one.
+![Autocomplete](./images/intellij/autocomplete.png)
+
+### Generate
+![Generate](./images/intellij/generate.png)
+Ctrl+N brings up a generate dialog, which can auto-generate
```

Comment:
if this is for the macOS scheme, isn't this cmd+N?

Reply:
Control is still a key for Mac keyboards, though command and option do most of the heavy lifting. Does it not work for you?

Reply:
yeah, I meant that on my mac keyboard cmd+N brings up the generate dialog, ctrl+N does something else